src/org/spdx/licensexml/LicenseXmlHelper.java

```diff
@@ -324,6 +339,28 @@ private static void appendOptionalText(Element element,
 		}
 	}
 
+	/**
+	 * @param element parent element
+	 * @return true if the element includes any phrasing content per https://www.w3.org/TR/2014/REC-html5-20141028/dom.html#phrasing-content-1
+	 */
+	private static boolean includesPhrasing(Element element) {
```

Comment:
This function should be `includesFlowControl`, and you'll want to update its comment, callers, etc.  This is bubbling up the `FLOW_CONTROL_ELEMENTS` rename to use the correct name throughout the new code.